QuasarHelios2015/src/org/team1540/quasarhelios/Clamp.java

```diff
@@ -60,7 +60,6 @@ public static void setup() {
         EventStatus zeroEncoder = new EventStatus();
         needsAutoCalibration.setFalseWhen(zeroEncoder);
         FloatInputPoll encoder = Igneous.makeEncoder(10, 11, true, zeroEncoder);
-        Igneous.startAuto.send(zeroEncoder);
```

Comment:
We don't always start with it at the top. And even when we do - the autocalibrator is in charge of this.